core/data/event.go

```diff
@@ -219,23 +191,22 @@ func eventHandler(w http.ResponseWriter, r *http.Request) {
 		loggingClient.Info("Posting Event: " + e.String())
 
 		// Get device from metadata
-		deviceFound := true
-		// Try by ID
-		d, err := mdc.Device(e.Device)
+		d, err := mdc.CheckForDevice(e.Device)
 		if err != nil {
-			// Try by name
-			d, err = mdc.DeviceForName(e.Device)
-			if err != nil {
-				deviceFound = false
+			loggingClient.Error(fmt.Sprintf("error checking device %s %v", e.Device, err))
+			switch err := err.(type) {
+				case types.ErrNotFound://ignore
+			default: //return an error on everything else.
+			    loggingClient.Debug(fmt.Sprintf("error of type %v", err))
```

Comment:
This log can be removed, we already logged 4 lines before

Reply:
removed with above changes